examples/server/tests/features/truncation.feature

```diff
@@ -0,0 +1,44 @@
+# run with: ./tests.sh --no-skipped --tags truncation
+@trucation
```

Comment:
I almost couldn't figure out why I couldn't get this new test to run.

```suggestion
@truncation
```

Finally turned to Claude who helped me track down this typo. Once corrected, I was able to run this test scenario with the command `./tests.sh --no-skipped --tags truncation` as-given.